apps/src/javalab/components/JavalabDropdown.jsx

```diff
@@ -0,0 +1,76 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+
+/**
+ * Component for a list of links that appears as a dropdown.
+ * This component only includes the dropdown itself,
+ * not the button that opens/closes the dropdown.
+ */
+class JavalabDropdown extends Component {
+  static propTypes = {
+    className: PropTypes.string,
+    children: props => {
+      React.Children.map(props.children, child => {
+        if (child.type !== 'a') {
+          throw new Error('only accepts children of type <a/>');
+        }
+        if (!child.props.href && !child.props.onClick) {
+          throw new Error('each child must have an href or onclick');
+        }
+      });
+    },
```

Comment:
Probably outside the scope of this PR, but any way we can share this validation function across other components? I think InlineDropdownMenu and DropdownButton both have the same or similar prop validation.

Reply:
Yeah I think it's out of scope for this, but it would be nice to have a unified dropdown component across the whole site.